fd-send-recv-tuntap.opam

```diff
@@ -14,10 +14,11 @@ tags: [
   "org:mirage"
```

Comment:
It's interesting that we have a separate opam file for the tuntap tests, but I guess this is the best way, since optional dependencies are harmful.